src/com/javabasic/lesson14oop/HomeWorkProcessor.java

```diff
@@ -0,0 +1,39 @@
+package com.javabasic.lesson14oop;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeWorkProcessor {
+
+    public static void main(String[] args) throws InterruptedException, IOException {
+
+        // Task №1
+        List<RealEstateTax> taxes = new ArrayList<>();
+
+        RealEstateTax residentialRealEstateTax = new ResidentialRealEstateTax(
+                28, 73, 23
+        );
+        RealEstateTax commercialRealEstateTax = new CommercialRealEstateTax(
+                21, 55, 120
+        );
+        RealEstateTax industrialRealEstateTax = new IndustrialRealEstateTax(
+                13, 12
+        );
+
+        taxes.add(residentialRealEstateTax);
+        taxes.add(commercialRealEstateTax);
+        taxes.add(industrialRealEstateTax);
+
+        for (RealEstateTax tax : taxes) {
+            System.out.println("Tax for " + tax.getClass().getSimpleName() + ": " + tax.getRealEstateTax());
+        }
+
+
+        // Task №2
+        CoffeeMachine coffeeMachine = new CoffeeMachine();
```

Comment:
Split these two tasks into separate packages. Each one should have its own main class and method. It's difficult to review and navigate through code when one task is mixed with other. Always use different packages for each task.

Reply:
Done